mcp/sensors/dsp1760.c

```diff
@@ -382,6 +469,13 @@ static int dsp1760_check_baud(int fd, int gyrobox, struct termios* term)
 }
 
 
+/**
```

Comment:
remove magic number 4103 from line 464

Reply:
I'll defend this, because I put it there...it's explained in a comment directly adjacent to its use, so it's less magic than it could be. It could have a variable name assigned to it, sure, but it wouldn't add any more clarity than the comment in this case. This really is a case where we need to compare a result to a specific constant number.

Reply:
Ugh, since it appears twice, I guess it really should be replaced with a named variable.
Do this at the top of the file, and replace the magic number with `BAUD_921600_MACRO`:

```c
// On newer systems, for querying the baudrate with cfgetispeed,
// B921600 is macro'd to 0010007 octal, or 4103 decimal.
// See e.g. /usr/include/asm-generic/termbits.h.
uint16_t BAUD_921600_MACRO = 4103;
```